pkg/controller/velerobackup/velerobackup_controller.go

```diff
@@ -223,14 +237,18 @@ func (r *ReconcileBackup) createVeleroBackupObject(namespace string, t metav1.Ti
 	backup := &velerov1.Backup{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      fmt.Sprintf("backup-%v-%v", namespace, timestamp),
-			Namespace: velerov1.DefaultNamespace,
+			Namespace: r.veleroNamespace,
 		},
 		Spec: velerov1.BackupSpec{
+			VolumeSnapshotLocations: []string{},
 			IncludedNamespaces: []string{
 				namespace,
 			},
 			ExcludedResources: defaultExcludedBackupResources,
 		},
+		Status: velerov1.BackupStatus{
```

Comment:
What is the purpose of including the `Status`? For most resource types, the create of the resource will clear out the status.

Reply:
Yeah, this was something new. So apparently the most recent version of Velero has a validation hook that errors without this. I discovered this during my manual tests and adding this line made it work.